ui/src/shared/components/CodeSnippet.scss

```diff
@@ -4,27 +4,32 @@
 */
 
 .code-snippet {
-  margin: $ix-marg-a 0;
-  border-radius: $ix-radius;
+  border-radius: $cf-radius;
   overflow: hidden;
+  border: $cf-border solid $g2-kevlar;
+  background-color: $g2-kevlar;
+  margin-bottom: $cf-marg-b;
 }
 
 .code-snippet--scroll {
   min-height: 26px !important;
-  background-color: $g4-onyx;
+  max-height: 200px;
+  border-radius: $cf-radius-sm;
+  overflow: hidden;
```

Comment:
If we set the overflow to hidden here, aren't we going to confine the DapperScrolls to an artificial div size that won't render all the data?

Reply:
The scroll area will grow to fit the size here so no weird behavior happening. The hidden overflow here ensures the corners are always rounded